apps/fluent-tester/src/TestComponents/Separator/SeparatorTest.tsx

```diff
@@ -1,6 +1,7 @@
 import * as React from 'react';
-import { Text } from 'react-native';
-import { Button, Separator } from '@fluentui/react-native';
+import { Separator } from '@fluentui-react-native/separator';
+import { Button } from '@fluentui-react-native/experimental-button';
```

Comment:
Please use ButtonV1 from @fluentui-react-native/button instead, experimental-button is deprecated.

Reply:
updated.